analysis/study_definition.py

```diff
@@ -376,148 +375,128 @@
 
 measures = [
   
-  #  Monthly rates - whole population
-  
-  ## Any opioid
+  #  Monthly rates #
```

Comment:
for some of your variables (e.g. carehome_codes on line 185) you are returning `binary_flag` with `return_expectations` for date. It won't cause any errors but is unnecessary.

Reply:
Ok - I thought I was telling it over what time period to look for the codes but on reflection I get that `return_expectations` is just about what the dummy data look like so it isn't used in that way.